ra_aid/agent_utils.py

```
@@ -38,7 +38,7 @@
 from ra_aid.agents_alias import RAgents
 from ra_aid.config import DEFAULT_MAX_TEST_CMD_RETRIES, DEFAULT_MODEL
 from ra_aid.console.formatting import print_error
-from ra_aid.console.output import print_agent_output
+from ra_aid.console.output import cpm, print_agent_output
```

Ruff (0.8.2)

`ra_aid.console.output.cpm` imported but unused

Remove unused import: `ra_aid.console.output.cpm`

(F401)